### PR TITLE
[DOCS-5850] Additional properties for Aspose and Docusign connectors

### PR DESCRIPTION
**Pull request by m-hulbert**

### CODE REVIEW

**Review by pmartinezga — APPROVED** [review]

File: docs/modeling/connectors/ootb/aspose.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ The following are the parameters that can be passed to the Aspose connector as i
 | `metadata` | The metadata to use in the template. If no value for `metadata` is passed then the inbound variable map will be used in the template | JSON | No |
 | `outputFormat` | Format to save the output document in. Values are `DOCX` or `PDF` | String | Yes |
 | `outputFileName` | The name of the generated file | String | No |
+| `targetFile` | The node ID of an existing file to overwrite with the generated document. | String | No |
```
Comment (mmartinadanx) [inline docs/modeling/connectors/ootb/aspose.md:56]: This should not be a nodeId, it's actually a 'file' object

File: docs/modeling/connectors/ootb/aspose.md [hunk 2]
```diff
@@ -53,6 +53,7 @@ The following are the parameters that can be passed to the Aspose connector as i
...
 | `parentFolder` | The node ID of the folder to store the generated document in. If this value is set, the generated document will be output here and not to the default process instance folder for the process instance | String | No |
```
Comment (mmartinadanx) [inline docs/modeling/connectors/ootb/aspose.md:57]: This should not be a node id, it's actually a 'folder' object

File: docs/modeling/connectors/ootb/docusign.md [hunk 3]
```diff
@@ -62,6 +62,7 @@ The following are the parameters that can be passed to the DocuSign connector as
 | `timeout` | The time to wait for the document to be signed expressed in seconds | Integer | No | 
 | `parentFolder` | The node ID of the folder to store the signed document in. If this value is set, the generated document will be output here and not to the default process instance folder for the process instance | String | No |
 | `outputFileName` | The name of the signed document | String | No |
+| `targetFile` | The node ID of an existing file to overwrite with the signed document. | String | No |
```
Comment (mmartinadanx) [inline docs/modeling/connectors/ootb/docusign.md:65]: This should not be a node id, it's actually a 'file' object

File: docs/modeling/connectors/ootb/docusign.md [hunk 4]
```diff
@@ -62,6 +62,7 @@ The following are the parameters that can be passed to the DocuSign connector as
 | `timeout` | The time to wait for the document to be signed expressed in seconds | Integer | No | 
 | `parentFolder` | The node ID of the folder to store the signed document in. If this value is set, the generated document will be output here and not to the default process instance folder for the process instance | String | No |
```
Comment (mmartinadanx) [inline docs/modeling/connectors/ootb/docusign.md:63]: This should not be a node id, it's actually a 'folder' object

**Pull request merged**